src/services/authenticator.js

```diff
@@ -51,29 +50,17 @@ function Authenticator() {
     || 'Invalid token. Please enter your authentication token.';
 
   this.logout = async (opts = {}) => {
-    const path = `${os.homedir()}/.forestrc`;
-
-    return new P((resolve, reject) => {
-      fs.stat(path, (err) => {
```

Comment:
Out of curiosity, why has it been removed?

Reply:
it was doing the job 'isLoggedIn?' by just checking the presence of the `.forestrc` file.
Now we rely on the `getVerifiedToken`because:
-it's same code than we use to evaluate at login-in
-it's smarter (validate the token) 
-it's more simple (no promise)